packages/hub/services/in-app-purchases.ts

```diff
@@ -1,8 +1,56 @@
+import config from 'config';
+import fetch from 'node-fetch';
```

Comment:
Do we need to import `fetch` here? We assign `global.fetch` in `main.ts`

Reply:
news to me, thanks!